Godeps/_workspace/src/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -148,6 +148,9 @@ type KubeletServer struct {
 
 	KubeApiQps   float32
 	KubeApiBurst int
+
+	// Pull images one at a time.
+	SerializeImagePulls bool
```

Comment:
Release note with description on how to turn off is here:
https://github.com/openshift/openshift-docs/issues/1131#issuecomment-153485270